lib/matplotlib/animation.py

```diff
@@ -686,28 +688,35 @@ def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
 
         *fps* is the frames per second in the movie. Defaults to None,
         which will use the animation's specified interval to set the frames
-        per second.
+        per second. This argument is only valid if *writer* is not a
+        :class:`MovieWriter` instance.
```

Comment:
I thought about that initially, but didn't because it would change order of arguments away from the function signature--should have just ignored that impulse.